examples/pyfluent-workflow/main.py

```diff
@@ -0,0 +1,220 @@
+# # Integrating PyFluent with PyWorkbench 
+# This example shows how to use the PyFluent workflow together with PyWorkbench - (Python client scripting for Ansys Workbench).
+#
+# This example sets up and solves a three-dimensional turbulent fluid flow
+# and heat transfer problem in a mixing elbow, which is common in piping
+# systems in power plants and process industries. Predicting the flow field
+# and temperature field in the area of the mixing region is important to
+# designing the junction properly.
+#
+# This example uses Pyfluent settings objects API's.
+#
+# ## Problem description
+#
+# A cold fluid at 20 deg C flows into the pipe through a large inlet. It then mixes
+# with a warmer fluid at 40 deg C that enters through a smaller inlet located at
+# the elbow. The pipe dimensions are in inches, and the fluid properties and
+# boundary conditions are given in SI units. Because the Reynolds number for the
+# flow at the larger inlet is ``50,800``, a turbulent flow model is required.
+
+# ## Performed required imports
+# Performing essential imports for Ansys Workbench, Fluent Pythonic Interface and for downloading examples data.
+
+import os
+import pathlib
+from ansys.workbench.core import launch_workbench
+import ansys.fluent.core as pyfluent
+from ansys.fluent.core import examples
+
+# ## Specify client and server directories with launch WB service.
+
+workdir = pathlib.Path("__file__").parent
+
+server_workdir = workdir / 'server_workdir'  
+server_workdir.mkdir(exist_ok=True)  
+
+wb = launch_workbench(release="241", server_workdir=str(server_workdir.absolute()), client_workdir=str(workdir.absolute()))
+
+# ## Download the input file from example data and upload to server directory.
+
+import_filename = examples.download_file("mixing_elbow.msh.h5", "pyfluent/mixing_elbow")
+wb.upload_file(import_filename)
+
+# ## Generate a "FLUENT" System using Ansys Workbench Scripting API (used for Journaling) and parse it to the PyWorkbench API.
+
+export_path = 'wb_log_file.log'
+wb.set_log_file(export_path)
+wb.run_script_string('template1 = GetTemplate(TemplateName="FLUENT")', log_level='info')
+wb.run_script_string('system1 = template1.CreateSystem()')
+
+
+# ## Launch Fluent & Connect to Fluent
+# Launch Fluent as server with PyWorkbench API and and connect to Pyfluent session
+
+server_info_file = wb.start_fluent_server(system_name = "FLU")
+fluent_session = pyfluent.connect_to_fluent(server_info_file_name= server_info_file)
+
+
+# ## Import mesh and perform mesh check
+
+# Import the mesh and perform a mesh check, which lists the minimum and maximum
+# x, y, and z values from the mesh in the default SI units of meters. The mesh
+# check also reports a number of other mesh features that are checked. Any errors
+# in the mesh are reported. Ensure that the minimum volume is not negative because
+# Fluent cannot begin a calculation when this is the case.
+
+import_filename = os.path.join(server_workdir.absolute(), 'mixing_elbow.msh.h5')
```

Comment:
You can get WB server working directory this way instead:
`server_dir = wb.run_script_string("wb_script_result=GetServerWorkingDirectory()")`